src/unistd.rs

```diff
@@ -2136,3 +2146,424 @@ mod setres {
         Errno::result(res).map(drop)
     }
 }
+
+
+#[derive(Debug, Clone, PartialEq, Default)]
```

Comment:
Attributes should got under the doccomments.

Reply:
done.